src/content/blog/posts/post-1.md

```diff
@@ -1,18 +1,193 @@
+- [excalidraw](https://excalidraw.com/) (and [VS Code extension](https://marketplace.visualstudio.com/items?itemName=pomdtr.excalidraw-editor)) for creating editable diagrams and whiteboard sketches which can be saved as code
+- [marp](https://marp.app/) for creating slide decks in markdown which can be saved as code
+
+Those internal to MOJ can access various use cases linked in the Excalidraw whiteboard below:
+
+<a href="https://excalidraw.com/#url=https://raw.githubusercontent.com/ministryofjustice/data-and-analytics-engineering/add-github-blog/src/content/blog/posts/images/github-one-stop-shop.excalidraw.png"><img src="images/github-one-stop-shop.excalidraw.png" width="80%" height="80%"></a>
+
+This article lists some recommendations on how to use these tools as of February 2024. For more detailed and/or up-to-date instructions please refer to the links provided. If you have any comments and/or suggestions please post on the [GitHub Discussion](https://github.com/ministryofjustice/data-and-analytics-engineering/discussions/10).
+
+# Why store everything in GitHub?
+
+The tools discussed in this article can have less features and may involve a steeper learning curve than your existing setup. In addition, there aren't clear migration paths which means that unless you're starting from scratch it can be painful to move accross.
+
+However, there are various advantages which makes it worthwhile:
+
+### Searching
+
+Gathering all your code, tickets, documentation, and presentations in one place makes it super easy to search through everything, *if you can find it*. However, GitHub Code Search which was first released in [2008](https://github.blog/2021-12-15-a-brief-history-of-code-search-at-github/) used to [suck](https://github.com/isaacs/github/issues/908). GitHub rolled out a new code search engine and browser in [November 2022](https://github.blog/changelog/2022-11-09-introducing-an-all-new-code-search-and-code-browsing-experience/), finally enabling users to find relevant results at reasonable speed and turning GitHub into a viable developer hub.
+
+### Interoperability
+
+Using GitHub tools exclusively makes it easier to link and automate across your project artefacts. Hence it makes it easier to track, review and deploy changes as well as collaborate with your team and users.
+
+### Standardisation
+
+Treating project [deliverables as code](https://thenewstack.io/why-docs-as-code-should-be-part-of-your-dev-cycle/) means you can leverage the same practices that are applied to software development: version control, automation, reusability and maintainability. This leads to improved standardization and quality across your projects and enterprise.
+
+### Cost
+
+You company may save money by no longer needing licenses for redundant tools like JIRA and Atlassian. However, this benefit is less significant if your company still has to pay for licenses for teams that don't use GitHub and are unlikely to switch.
```

Comment:
As above – we should not reference specific tools here. We can say something along the lines of using fewer tools may allow you to save money.